packages/vscode-docker-registries/src/clients/GenericRegistryV2/GenericRegistryV2DataProvider.ts

```diff
@@ -85,6 +85,7 @@ export class GenericRegistryV2DataProvider extends RegistryV2DataProvider {
             registryPromptPlaceholder: vscode.l10n.t('http://localhost:5000'),
             usernamePrompt: vscode.l10n.t('Registry Username'),
             secretPrompt: vscode.l10n.t('Registry Password or Personal Access Token'),
+            connectedRegistries: this.authenticationProviders,

```

Comment:
Can you turn this into an array of URLs rather than pass the entire dictionary of URL -> auth provider?

Reply:
yeah that makes sense